src/pyosmeta/_version.py

```diff
@@ -0,0 +1,17 @@
+# file generated by setuptools_scm
+# don't change, don't track in version control
```

Comment:
This should go in `.gitignore`

Reply:
Thank you so so much, @ofek !! i just fixed this and pushed it up. i really appreciate your help.